az-cvm-vtpm/src/vtpm/mod.rs

```diff
@@ -126,6 +128,24 @@ impl Quote {
         let nonce = attest.extra_data().to_vec();
         Ok(nonce)
     }
+
+    pub fn verify_pcrs(&self) -> Result<bool, QuoteError> {
```

Comment:
this should probably be implemented to verify.rs, so we don't need crypto functions on if the `verifier` feature is disabled (on a TEE)